src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/IntegrationTest.kt

```diff
@@ -22,7 +22,7 @@ internal abstract class IntegrationTest(useCustomDriver: Boolean = false) : Flue
   private lateinit var logoutButton: FluentWebElement
 
   // The custom driver is for testing the spreadsheet download functionality only!
-  private val testDriver: WebDriver = if (useCustomDriver) CustomHtmlUnitDriver() else ChromeDriver(ChromeOptions().apply { addArguments("--headless") })
```

Comment:
Is this on purpose?

Reply:
Nope.  I took it out running locally and forgot to put back in.... ("Doh" in Homer Simpson voice).